packages/dendron-next-server/components/graph.tsx

```diff
@@ -38,6 +40,12 @@ const getCytoscapeStyle = (themes: any, theme: string | undefined) => {
     background-color: ${AntThemes[theme].graph.node._selected.color};
     color: ${AntThemes[theme].graph.node._selected.color};
   }
+  
+  .parent {
+    shape: diamond;
+    width: ${AntThemes[theme].graph.node.size * 1.25};
```

Comment:
how did we decide on the 1.25? should be a constant with a comment that describes reasoning

Reply:
1.25 is to compensate for the fact that diamond-shaped nodes appear smaller than circular ones. I'll refactor to be more specific